connect/support/timer.py

```diff
@@ -30,7 +33,12 @@ async def timer_wrapper(*args, **kwargs):
             result = func(*args, **kwargs)
 
         run_time = time.time() - start_time
```

Comment:
Would there be an advantage to using [perf counter](https://docs.python.org/3/library/time.html#time.perf_counter_ns)?

Reply:
I've seen both used for this use case, but I don't think there is a difference.  "perf_counter() should measure the real amount of time for a process to take, as if you used a stop watch."  But that is also what we do with time.time(), so I think they are equivalent.